pages/k8s/gpu-workers.md

```diff
@@ -13,45 +13,56 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
-**Charmed Kubernetes** supports GPU-enabled instances for applications that
-can use them. The `kubernetes-worker` application will automatically detect
-NVIDIA hardware and enable the appropriate support. This page describes
-recommended deployment and verification steps when using GPU workers with
-Charmed Kubernetes.
-
-### Deploying Charmed Kubernetes with GPU workers
-
-When deploying the Charmed Kubernetes bundle, you can use a YAML overlay file
-with constraints to ensure worker units are deployed on GPU-enabled machines.
-Because GPU support varies depending on the underlying cloud, this requires
-specifying a particular instance type.
-
-For example, when deploying to AWS, you may decide to use a `p3.2xlarge`
-instance from the available [AWS GPU-enabled instance types][aws-instance].
-Similarly, you could choose Azure's `Standard_NC6s_v3` instance from the
-available [Azure GPU-enabled instance types][azure-instance].
-
-NVIDIA updates its list of supported GPUs frequently, so be sure to cross
-reference the GPU included in a specific cloud instance against the
+**Charmed Kubernetes** can automatically detect GPU-enabled hardware and install
+required drivers from NVIDIA repositories. With the release of Charmed Kubernetes 1.29,
+the [NVIDIA GPU Operator charm][nvidia-gpu-operator] is available to simplify gpu
+software management and configuration.
+
+This page describes recommended deployment and verification steps for using GPU workers
+with Charmed Kubernetes.
+
+## Deploying Charmed Kubernetes with GPU workers
+
+GPU support varies depending on the underlying cloud, so you will need to determine a
+particular instance type for the `kubernetes-worker` application.
+
+For example, when deploying to AWS, you may decide to use a `p3.2xlarge` instance from
+the available [AWS GPU-enabled instance types][aws-instance].
+Similarly, you could choose an Azure `Standard_NC6s_v3` instance from the available
+[Azure GPU-enabled instance types][azure-instance].
+
+NVIDIA updates its list of supported GPUs frequently, so be sure to cross reference the
+GPU included in a specific cloud instance against the
 [Supported NVIDIA GPUs and Systems][nvidia-gpu-support] documentation.
 
-Example overlay files that set GPU worker constraints:
+When deploying the Charmed Kubernetes bundle, you can use a YAML overlay file to ensure
+application instance types and configuration are optimized for use in a GPU-enabled
```

Comment:
```suggestion
application instance types and configuration are optimised for use in a GPU-enabled
```

Reply:
grrrr, tried to sneak this one by you :)